src/main/java/mekanism/common/registries/MekanismDataMapTypes.java

```diff
@@ -0,0 +1,14 @@
+package mekanism.common.registries;
+
+import mekanism.common.Mekanism;
+import mekanism.common.datamaps.MekaSuitAbsorption;
+import mekanism.common.registration.impl.DataMapTypeRegister;
+import net.minecraft.core.registries.Registries;
+import net.minecraft.world.damagesource.DamageType;
+import net.neoforged.neoforge.registries.datamaps.DataMapType;
+
+public class MekanismDataMapTypes {
+    public static final DataMapTypeRegister REGISTER = new DataMapTypeRegister(Mekanism.MODID);
+
+    public static final DataMapType<DamageType, MekaSuitAbsorption> MEKA_SUIT_ABSORPTION = REGISTER.registerSimple("mekasuit_absorption", Registries.DAMAGE_TYPE, MekaSuitAbsorption.CODEC);
```

Comment:
We probably want to expose this `DataMapType` to the API, unless there is some other way for addons and the like to potentially get it.

Reply:
`RegistryManager#getDataMap` can be used for that, though it's a bit annyoing.